### PR TITLE
Create pattern: OMIM_phenotypic_series

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
addresses #2133

### CODE REVIEW

File: src/patterns/dosdp-patterns/OMIM_phenotypic_series [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+vars:
+    disease: "'disease'"
+
+name:
+    text: '%s'
+    vars:
+      - disease
+
+pattern_iri: http://purl.obolibrary.org/obo/mondo/patterns/OMIM_phenotypic_series.yaml
```
Comment (cmungall) [inline src/patterns/dosdp-patterns/OMIM_phenotypic_series:32]: @matentzn should we write this or infer it?

File: src/patterns/dosdp-patterns/OMIM_phenotypic_series [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+- the OMIM subclasses must have acronym synonyms that are the parent syn + number, e.g. HPE1, HPE2
+- the primary label for the children should also be parent + {"type"} + number
+- the first member will usually have the same number local ID as the PS
+- the first member in OMIM usually has documentation that is pertinent to the parent PS
+- the members may(?) generally share high semantic similarity
+
+Examples: 
+[holoprosencephaly](http://purl.obolibrary.org/obo/MONDO_0016296) [OMIMPS:236100](https://omim.org/phenotypicSeries/PS236100), '3-M syndrome'(http://purl.obolibrary.org/obo/MONDO_0007477) [OMIMPS:236100](https://omim.org/phenotypicSeries/PS273750). 
+
+classes:
```
Comment (cmungall) [inline src/patterns/dosdp-patterns/OMIM_phenotypic_series:21]: @matentzn there is no equiv axiom for this. Shall we just have a generic subClassOf? Or leave this off altogether? Don't want to break docs

File: src/patterns/dosdp-patterns/OMIM_phenotypic_series [hunk 3]
```diff
@@ -0,0 +1,36 @@
+pattern_name: OMIM_phenotypic_series
+
+description: >-
+
+This patterrn is meant to be used for OMIM phenotypic series (OMIMPS), which are represented as grouping classes in Mondo. Note: 
+- every instance of this metaclass should be equivalent to (via annotated xref) to something in OMIMPS namespace
```
Comment (cmungall) [inline src/patterns/dosdp-patterns/OMIM_phenotypic_series:6]: @matentzn not urgent but to think about for future pattern calls. How do we encode this? Do we extend the dosdp formalism? Until it ends up incorporating half of shex? this is why I think we need to ultimately subsume within a more general framework

File: src/patterns/dosdp-patterns/OMIM_phenotypic_series (hunk 2 above)
Comment (cmungall) [reply src/patterns/dosdp-patterns/OMIM_phenotypic_series:21]: Formally these always have a NC-NC equiv axiom to the PS... but this is not encoded in the edit file (except as annotated xref). Hmmm..

**Review by cmungall — APPROVED** [review]
Good to go if @matentzn approves

File: src/patterns/dosdp-patterns/OMIM_phenotypic_series (hunk 1 above)
Comment (matentzn) [reply src/patterns/dosdp-patterns/OMIM_phenotypic_series:32]: Always write!

File: src/patterns/dosdp-patterns/OMIM_phenotypic_series (hunk 2 above)
Comment (matentzn) [reply src/patterns/dosdp-patterns/OMIM_phenotypic_series:21]: I think we are stretching the scope of DOSDP if we try to encode this kind of logic.. I would suggest we start now seriously with the design of a pattern formalism from which the other ones can be generated; there then sky is the limit. Right now I would advocate no logical axiom in the pattern.

**Pull request merged**